### ISSUE TITLE
Can't figure out how to install BIOS version 4.12

### ISSUE THREAD

**Issue by andrew-skybound** [TheRandMan/Hackintosh---Dell-Chromebook-13-7310#32]
The guide says that MrChromebox 4.13 doesn't work, but how do I install 4.12?  The script on MrChromebox's website only installs the latest version of the bios, and according to [this](https://github.com/MrChromebox/firmware/issues/283) issue there's no way to install a previous version.

**Comment by TheRandMan** [comment]
Unfortunately, nothing can really be done about this until I (or somebody else!) can figure out why 4.13 is not working. I haven't had much of a chance to look into it but still hope to figure it out.

**Comment by arsfeld** [comment]
Turns out you can downgrade to version 4.12, although "not supported", which just means the script won't work for you as is. What I ended up doing is cloning the repo from https://github.com/MrChromebox/scripts and changing the `bmodels.json` file to use the older version, which the link is https://mrchromebox.tech/files/firmware/full_rom/coreboot_tiano-lulu-mrchromebox_20200604.rom

To actually run it though, the `firmware-util.sh` script downloads the latest version of the scripts, which is not what you want, so I just erased the lines that downloaded it and ran it manually. There might be an easier way.

It did work for me, but you should be careful, if you make a mistake you might end up flashing the wrong firmware.

**Comment by andrew-skybound** [comment]
I'm guessing that's a link to the UEFI full rom... Is it possible to flash RW_LEGACY instead?  Do you know the link to 4.12 RW_LEGACY, or could you point me in the right direction?

**Comment by andrew-skybound** [comment]
Ok, I checked out commit 9273784c of MrChromebox/scripts which was the last commit for version 4.12, as far as I can tell.  I flashed my bios, which is coming up as **SeaBIOS (version rel-1.11.0-54-g248459e-MrChromebox-2018.09.12)** when I boot.  Unfortunately, it still won't boot the macOS installer USB.  After selecting boot device 2 (USB disk) the drive flashes it's LED about 10 times and then just sits where it is.  I've put a debug build of opencore on the USB drive, but it doesn't look like there are any logs being written, and no matter how long I wait for it to boot, nothing happens.

I can boot ubuntu from a usb drive, so booting from a USB disk is possible, just not the installer USB.  Have I followed all the directions?  Yes, twice, as carefully as possible.  I tried it first creating the installer from Windows, and the second time created it using a mac.

Any suggestions?  Or do I need to install the UEFI full rom instead?
